modules/ROOT/pages/design-delete-project.adoc

```diff
@@ -0,0 +1,23 @@
+= Delete a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can delete an API-specification project or an API-fragment project.
+
+[NOTE]
```

Comment:
Not sure this needs to be called out as a Note, esp. since we following it with another notice.
See what the style reference has to say about notice formats.